### ISSUE TITLE
Billboard 200 not returning any charts prior to 11/18/1989

### ISSUE THREAD

**Issue by azhadsyed** [guoguo12/billboard-charts#71]
Hi all, when running the below code to try and print the Billboard 200 on any chart on or before 11/18/1989, I get this message:

```
import billboard
def chart_to_dict(chart):
    output = []
    for entry in chart.entries:
        row = {'title':entry.title,
               'artist':entry.artist,
               'rank':entry.rank}
        output.append(row)
    return output

class ChartArchive:
    def __init__(self, chart_name):
        self.chart_name = chart_name
        self.charts = []
    def one_off(self, dates):
        for date in dates:
            chart = billboard.ChartData(self.chart_name, date)
            d = {'date': chart.date, 'chart': chart_to_dict(chart)}
            self.charts.append(d)

one_off_date = ['1989-11-18']
one_off_name = 'billboard-200'
one_off_chart = ChartArchive(one_off_name)
one_off_chart.one_off(one_off_date)
one_off_chart.charts

```

```
AttributeError                            Traceback (most recent call last)
~\Anaconda3\lib\site-packages\billboard.py in _parseNewStylePage(self, soup)
    290             try:
--> 291                 artist = getEntryAttr("span.chart-element__information__artist") or ""
    292             except:

~\Anaconda3\lib\site-packages\billboard.py in getEntryAttr(selector)
    281             def getEntryAttr(selector):
--> 282                 return entrySoup.select_one(selector).text.strip()
    283 

AttributeError: 'NoneType' object has no attribute 'text'

During handling of the above exception, another exception occurred:

BillboardParseException                   Traceback (most recent call last)
<ipython-input-9-38c654d4ba80> in <module>
     23 one_off_name = 'billboard-200'
     24 one_off_chart = ChartArchive(one_off_name)
---> 25 one_off_chart.one_off(one_off_date)
     26 one_off_chart.charts

<ipython-input-9-38c654d4ba80> in one_off(self, dates)
     16     def one_off(self, dates):
     17         for date in dates:
---> 18             chart = billboard.ChartData(self.chart_name, date)
     19             d = {'date': chart.date, 'chart': chart_to_dict(chart)}
     20             self.charts.append(d)

~\Anaconda3\lib\site-packages\billboard.py in __init__(self, name, date, fetch, max_retries, timeout)
    150         self.entries = []
    151         if fetch:
--> 152             self.fetchEntries()
    153 
    154     def __repr__(self):

~\Anaconda3\lib\site-packages\billboard.py in fetchEntries(self)
    369 
    370         soup = BeautifulSoup(req.text, "html.parser")
--> 371         self._parsePage(soup)
    372 
    373 

~\Anaconda3\lib\site-packages\billboard.py in _parsePage(self, soup)
    349             self._parseOldStylePage(soup)
    350         else:
--> 351             self._parseNewStylePage(soup)
    352 
    353     def fetchEntries(self):

~\Anaconda3\lib\site-packages\billboard.py in _parseNewStylePage(self, soup)
    292             except:
    293                 message = "Failed to parse artist"
--> 294                 raise BillboardParseException(message)
    295 
    296             if artist == "":

BillboardParseException: Failed to parse artist
```

To confirm, the code above executes successfully when given the date 11/25/1989. Has anyone encountered this? Feedback is appreciated.

**Comment by gvant** [comment]
I am also having this issue. But the chart seems to work fine before 10/15/1988.

**Comment by gvant** [comment]
Okay so I now know what the issue is, and I fixed it.

An album named Silhouette was released by Kenny G that charted between October 1988 and November 1989, which is when the code stops working. This is because Kenny G's name is somehow completely missing for this entry (Billboard's mistake). 

**the fix**: Go to wherever your billboard.py code is. It's going to look something like \lib\site-packages\billboard.py or \lib\python3.7\site-packages.

Find lines 293 and 294 and erase them, and write this instead (you should be writing it under "except:", and remember to indent):
artist  = "Kenny G" 

This issue happens again in the late 60s for a few dates, but just repeat that process with the correct artist and change it back after. 

Hope this helped!

**Comment by guoguo12** [comment]
Fixed in 6.1.2. Thank you both!
